app/services/call_tool/call_status_updater.rb

```diff
@@ -4,24 +4,24 @@ class << self
       def start!(call)
         return unless call.unstarted?
         call.started!
-        CallEvent::Update.publish(call)
+        CallEvent::Update.publish(call) if call.member.present?
       end
 
       def connect!(call)
         return unless call.started?
         call.connected!
-        CallEvent::Update.publish(call)
+        CallEvent::Update.publish(call) if call.member.present?
       end
 
       def update!(call, params)
         call.update!(params)
-        CallEvent::Update.publish(call)
+        CallEvent::Update.publish(call) if call.member.present?
       end
 
       def new_member_call_event!(call, params)
         call.member_call_events << params
         call.save!
-        CallEvent::Update.publish(call)
+        CallEvent::Update.publish(call) if call.member.present?
```

Comment:
Do you think it might make sense to move this check to `CallEvent::Update.publish` so it's just there once?